app/src/main/java/com/example/dictionary/main/UserInteraction.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.dictionary.main
+
+sealed class UserInteraction {
```

Comment:
just try for fun, sealed interface :) and read about it..it's a better choice for the interaction

Reply:
done